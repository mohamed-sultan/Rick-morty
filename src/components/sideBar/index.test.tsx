import * as React from "react";
import TestRenderer from "react-test-renderer";
import SideBar from "./index";

describe("SideBar snapshot testing", () => {
  it("renders correctly", () => {
    const tree = TestRenderer.create(<SideBar />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
