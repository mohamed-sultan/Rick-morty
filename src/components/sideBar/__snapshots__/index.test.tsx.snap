// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideBar snapshot testing renders correctly 1`] = `null`;
